src/Login/Login.js

```diff
@@ -0,0 +1,39 @@
+import { gql, useQuery } from '@apollo/client'
+import React, { Component } from 'react'
+import { useHistory } from 'react-router-dom'
+
+const USER_LOGIN = gql`
+  query users($name: name, $password: password) {
+    user(name: $name, password: $password) {
+      id
+    }
+  }
+`;
+
+function Login() {
+  let input = {
+    password: '',
+    username: ''
+  }
+  let history = useHistory();
+  let handleInput = (e) => {
+    input[e.target.name] = e.target.value
+  }
+
+  let login = (e) => {
+    e.preventDefault();
+    history.push('/')
+  }
+
+  return (
+    <section>
+      <form>
+        <input type="text" placeholder="username" name="username" onInput={handleInput}/>
+        <input type="password" placeholder="password" name="password" onInput={handleInput}/>
```

Comment:
Not important but do you know a way that the user would be able to switch from types 
if a user would prefer to not see the dot when logging in

Reply:
So that a user could read what they typed in? Yeah! We could change the `input` `type` to conditionally be `password`  or `text`. The form could have a button that toggled that property back-and-forth.